module-install-4.ks.in

```diff
@@ -8,11 +8,10 @@
 %ksappend repos/default.ks
 %ksappend common/common_no_payload.ks
 
-module --name=cri-o --stream=1.11
+module --name=cri-o --stream=nightly
```

Comment:
I don't understand this test. Why do you install `avocado`, but enable `cri-o`?

Reply:
Looks like a typo - fixed. :)